cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4CareGapsService.java

```diff
@@ -320,25 +315,18 @@ protected <T extends Resource> T addConfiguredResource(Class<T> resourceClass, S
 
     protected List<Measure> ensureMeasures(List<Measure> measures) {
         measures.forEach(measure -> {
-            if (!measure.hasScoring()) {
-                ourLog.info("Measure does not specify a scoring so skipping: {}.", measure.getId());
-                measures.remove(measure);
-            }
             if (!measure.hasImprovementNotation()) {
-                ourLog.info("Measure does not specify an improvement notation so skipping: {}.", measure.getId());
-                measures.remove(measure);
+                ourLog.info(
```

Comment:
thought: This is the type of validation we should push upstream. Then we can give specific feedback in an OperationOutcome about issues with the Measure. In general, we _don't_ want to be lenient with these type of requirements. In this particular case, the improvement notation flips the meaning of the Measure. Good means bad, bad means good.